sessionctx/varsutil/varsutil.go

```diff
@@ -38,7 +38,6 @@ func GetSessionSystemVar(s *variable.SessionVars, key string) (string, error) {
 	if err != nil {
 		return "", errors.Trace(err)
 	}
-	s.Systems[key] = gVal
```

Comment:
Will this hurt the performance?

Reply:
Just remove a line.
This has nothing to do with performance.

Reply:
If the user gets the session variable multiple times, it will get global every time.

Reply:
@coocood 
It will be done in `SetSessionSystemVar` here:
https://github.com/pingcap/tidb/pull/5528/files#diff-cfa73fed62dec053d4e17a8f3605b708L172